doc/nrf/protocols/thread/certification.rst

```diff
@@ -177,3 +180,48 @@ This is valid only for Nordic Semiconductor development kits with a J-Link virtu
 
 To add an nRF52840 DK, drag the nRF52840 DK and drop it on the test bed configuration page.
 After that, the device is configured and the :ref:`proper baud rate (115200) <test_and_optimize>` and COM port are set.
+
+.. _ug_thread_build_report:
+
+OpenThread build report
+***********************
+
+The OpenThread build report contains information about the current |NCS| and OpenThread revisions, Thread feature set, Thread library path and changes in the :ref:`nrfxlib:nrfxlib` repository in comparison with the latest |NCS| release.
+The report is generated to the output console log, and stored as an additional build artefact in the application build directory.
+
+To enable building the Thread report set the :kconfig:option:`CONFIG_OPENTHREAD_REPORT` kconfig option to ``y``.
+To disable generating the build artefact set the :kconfig:option:`CONFIG_OPENTHREAD_REPORT_BUILD_ARTEFACT` kconfig option to ``n``.
+
+By default the report artefact name is set to the :file:`ot_report.txt`, but you can specify your own name by setting the :kconfig:option:`CONFIG_OPENTHREAD_REPORT_BUILD_ARTEFACT_NAME` kconfig value to the new one.
+
+While building the application using the :ref:`nrfxlib:ot_libs` you will see the following logs in your build console:
+
+.. code-block::
+
+   ################### OPENTHREAD REPORT ###################
+   + Target device: nrf52840
+   + Thread version: v1.3
+   + Feature set: master
+   + OpenThread Library from nrfxlib: openthread/lib/cortex-m4/soft-float/v1.3/master/
+   + OpenThread NCS revision: thread-reference-20230706-755-ga759a4a09
+   + NCS revision: v2.7.0-882-g3097ca5d36
+   ###################        END        ###################
```

Comment:
Can we have an example presenting how the potential changes between certified and current nrfxlib look like? I think it would be nice to show where user should look for such information.

Reply:
I've added such information, please take a look.